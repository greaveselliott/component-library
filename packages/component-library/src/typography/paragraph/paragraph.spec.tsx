import React from 'react';
import { shallow } from 'enzyme';

import Paragraph from './paragraph.component';

test('Paragraph component renders correctly', () =>
  expect(shallow(<Paragraph>abc</Paragraph>)).toMatchSnapshot());
