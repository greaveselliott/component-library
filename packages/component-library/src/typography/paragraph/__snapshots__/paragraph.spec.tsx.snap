// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph component renders correctly 1`] = `
<p
  className="paragraph"
>
  abc
</p>
`;
